src/Microsoft.Build.Utilities.ProjectCreation/PackageFeed.File.cs

```diff
@@ -81,6 +81,19 @@ public PackageFeed ContentFileText(string relativePath, string contents, NuGetFr
             return FileText(Path.Combine("contentFiles", contentFilePath), contents);
         }
 
+        /// <summary>
+        /// Adds a custom file to a package.
+        /// </summary>
+        /// <param name="relativePath">The relative path of the text file within the package.</param>
```

Comment:
Looks sufficient to me. We might wanna remove the `text` in the parameter doc as this file can be any file, right?